### ISSUE TITLE
PHP Fatal error:  Interface 'Doctrine\Common\EventSubscriber' not found 

### ISSUE THREAD

**Issue by umpirsky** [docteurklein/event-store#1]
```
$ php ./example/shop.php 
PHP Fatal error:  Interface 'Doctrine\Common\EventSubscriber' not found in /home/umpirsky/Projects/sylius.dev/event-store/example/_base.php on line 111
PHP Stack trace:
PHP   1. {main}() /home/umpirsky/Projects/sylius.dev/event-store/example/shop.php:0
PHP   2. require() /home/umpirsky/Projects/sylius.dev/event-store/example/shop.php:5
```

**Comment by cursedcoder** [comment]
`composer require doctrine/common`

**Comment by umpirsky** [comment]
Examples should work out of the box, at least with dev deps.

**Comment by docteurklein** [comment]
ho yes, I forgot to add this dep, I was in the train, working offline :/

Unfortunately, composer is not able to generate a valid psr-0 autoloader without an internet connection.

And thanks for pointing this out :)

**Comment by docteurklein** [comment]
closed by 318ff11ff43bf422245669a09c366f1ca6368aa0
